packages/ifpack2/src/Ifpack2_IlukGraph.hpp

```diff
@@ -211,13 +214,18 @@ template<class GraphType>
 IlukGraph<GraphType>::
 IlukGraph (const Teuchos::RCP<const GraphType>& G,
            const int levelFill,
-           const int levelOverlap)
+           const int levelOverlap,
+           const double overalloc)
   : Graph_ (G),
     LevelFill_ (levelFill),
     LevelOverlap_ (levelOverlap),
+    Overalloc_ (overalloc),
     NumMyDiagonals_ (0),
     NumGlobalDiagonals_ (0)
-{}
+{
+  TEUCHOS_TEST_FOR_EXCEPTION(Overalloc_ <= 1., std::runtime_error,
```

Comment:
Overalloc is ignored for level 0.  Allowing setting Overalloc to 1 would allow an infinite loop where the resizing never has effect.  If you prefer, I can allow overalloc=1, and then always actually use (overalloc+0.1) or similar.